package.json

```diff
@@ -3,7 +3,8 @@
 	"author": "Stefan Thomas <justmoon@members.fsf.org> (http://www.justmoon.net)",
 	"version": "3.0.2",
 	"description": "Port of jQuery.extend for node.js and the browser",
-	"main": "index",
+	"main": "index.cjs",
```

Comment:
`.js`, not `.cjs`, should be used for CJS for backwards compatibility. This is a potential breaking change.

Reply:
I've reverted this change